.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "tamia",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": "error",
+    "no-shadow": 0,
```

Comment:
Comparing with the .eslintrc.json file in .eslintrc.json repo, this setting has been added here.

Which effect does it have in the eslint rules?

Reply:
Same reasoning it was added [here](https://github.com/telefonicaid/iotagent-ul/pull/434/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090f) - the latest version of eslint complains about Telefonica style callbacks in the code (it wants the  `callback` variable name to be unique) hence the *shadowed variable name* rule must be dropped. This is only to the updates to a more modern **Eslint**

Reply:
Thanks for the clarification.

So, I understand that `"no-shadow": 0` is the desired setting and the repos not using it (if any) should be fixed. Out of the scope of this PR, of course.

NTC